pictureflow/transform/fit.py

```diff
@@ -0,0 +1,47 @@
+from pictureflow.core import Image, Node
+
+import cv2
+import numpy as np
+
+
+class FitToSize(Node):
+
+    _input_type = Image
+    _output_type = Image
+
+    def __init__(self, parent, tgt_size, id='size_fit'):
```

Comment:
You could have a 'best fit' or 'strict' param ... if best_fit is True, and min_x < 0 set it to zero. If you want it strict, still make it crash. Making it flexible and still crash in your case when you want it to crash